projects/showcase/src/app/app.component.spec.ts

```diff
@@ -13,7 +13,7 @@ import { ShowcaseModule } from './showcase/showcase.module';
 import { APP_BASE_HREF } from '@angular/common';
 
 describe('AppComponent', () => {
-	beforeEach(async(() => {
+	beforeEach(async () => {
 		TestBed.configureTestingModule({
```

Comment:
In the other cases, there is an await before the configureTestingModule, but not here.
Is this correct?